runtime/compiler/optimizer/EstimateCodeSize.hpp

```diff
@@ -85,6 +85,7 @@ class TR_EstimateCodeSize
    bool isLeaf()                       { return _isLeaf; }
 
    int32_t getNumOfEstimatedCalls()    { return _numOfEstimatedCalls; }
+   bool isInlineable(TR_CallStack *, TR_CallSite *callsite);
```

Comment:
Doxygen please since this has now been exposed....

Reply:
Added doxygen documentation.